src/main/interface/msp_box.c

```diff
@@ -281,27 +281,55 @@ int packFlightModeFlags(boxBitmask_t *mspFlightModeFlags)
     boxBitmask_t boxEnabledMask;
     memset(&boxEnabledMask, 0, sizeof(boxEnabledMask));
 
+    // copy ARM state
+    if (ARMING_FLAG(ARMED))
```

Comment:
I know it's just copy / paste, but can you add a block to this? Good coding practices _and_ Betaflight coding standards.